browser/src/canvas/sections/CommentSection.ts

```diff
@@ -1110,6 +1116,43 @@ export class Comment extends CanvasSectionObject {
 			brElements[brElements.length-1].remove();
 	}
 
+	private getSaveButton(): HTMLButtonElement | null {
+		return this.sectionProperties.nodeModify.querySelector('input[type="button"][id^="annotation-save-"]');
```

Comment:
maybe instead of querying it everytime just store them as a property like nodeModify on creation. same goes for the rest of query methods.